native-schema-registry/c/README.md

```diff
@@ -39,6 +39,16 @@ We use address,leak sanitizers to detect memory leaks and any potential issues d
 
 TBD
 
+## Develop
+
+### Add Class
+In order to add a class that is accessible by target language, these steps are required:
+1. Create the .h file in `include` directory
+2. Create the .i file in `src/swig` directory
+3. Create the .c file in `src` directory 
+4. Add the .i file to `glue_schema_registry_serde.i` file in `src/swig` directory
```

Comment:
Talk about tests as well?

Reply:
I agree but I still want to keep this as a checklist for now. This might be handy for people joining the project with no context.